app/models/concerns/vacancy_job_location_validations.rb

```diff
@@ -1,7 +0,0 @@
-module VacancyJobLocationValidations
-  extend ActiveSupport::Concern
-
-  included do
-    validates :job_location, presence: true
```

Comment:
Why get rid of the presence validation?

Reply:
I made a search for module VacancyJobLocationValidations, it is not included anywhere. In order to find the code that is not used, I have a look here:
https://sonarcloud.io/component_measures?id=DFE-Digital_teacher-vacancy-service&metric=coverage&view=list

Reply:
We already do `validates :job_location, presence: true` on the form model